### ISSUE TITLE
Does not work if gevent 1.3 is installed

### ISSUE THREAD

**Issue by miracle2k** [getlogbook/logbook#258]
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/logbook/__init__.py", line 14, in <module>
    from .base import (
  File "/usr/local/lib/python2.7/dist-packages/logbook/base.py", line 19, in <module>
    from logbook.concurrency import (greenlet_get_ident, thread_get_ident,
  File "/usr/local/lib/python2.7/dist-packages/logbook/concurrency.py", line 31, in <module>
    from gevent._threading import (Lock as ThreadLock,
ImportError: cannot import name RLock
```

**Comment by vmalloc** [comment]
I am not sure what to do about this specific bug. I'm tempted to just import `Lock` as `ThreadRLock` in this case, but this could cause deadlocks in use cases involving gevent. 

@miracle2k are you using Logbook with gevent while monkey-patched? If so I'd appreciate help testing the change out...

**Comment by miracle2k** [comment]
Unfortunately I am not using them together, afair gevent is just installed in my environment (possible due to gunicorn?) and that is enough to trigger the error.

**Comment by vmalloc** [comment]
Fixed in 1.3.1 (will be on pypi soon). Thanks for reporting!

**Comment by shidenggui** [comment]
In 1.3.1,  `_get_original('threading', 'get_ident')` will cause error in Python 2.7, should be `_get_ident`
```python
  File "/usr/local/lib/python2.7/dist-packages/logbook/base.py", line 19, in <module>
    from logbook.concurrency import (greenlet_get_ident, thread_get_ident,
  File "/usr/local/lib/python2.7/dist-packages/logbook/concurrency.py", line 34, in <module>
    thread_get_ident = _get_original('threading', 'get_ident')
  File "/usr/lib/python2.7/dist-packages/gevent/monkey.py", line 39, in get_original
    return _get_original(name, [item])[0]
  File "/usr/lib/python2.7/dist-packages/gevent/monkey.py", line 33, in _get_original
    values.append(getattr(module, item))
AttributeError: 'module' object has no attribute 'get_ident'
```

**Comment by vmalloc** [comment]
@shidenggui you are right. I fixed it and pushed 1.3.3 to correct it. I also added a specific Travis configuration for gevent+2.7 to spot such breakage in the future. Thanks!
